docs/components/select/elements.md

```diff
@@ -68,3 +68,21 @@ error.
 ```
 
 <api-table type=elements component="Select" />
+
+### Event handling
+The w-select component triggers the custom `change` event, on the element level, when the dropdown select element changes value.
+
+When using the element in Lit, you can listen to the event in the standard way:
+
+```js
+html`<w-select label="Berries" auto-focus @change=${e => handleChange(e)}>`
+```
+
+The selected dropdown value is available on the `detail` field of the event (event.detail).
+
+To use the event in other frameworks, such as Svelte, an event handler must be added manually:
```

Comment:
Good point! This already works in the React implementation, and was added here because of a request where it was used in Svelte. Let me know if I should add some more comments.

Reply:
aah I see! Hmm, if it was requested in Svelte, then maybe it is fine to show this example 🤔 👍